### ISSUE TITLE
the difference between load_from and pretrained

### ISSUE THREAD

**Issue by cs21203au22** [open-mmlab/mmaction2#1829]
Hi Team,
There seems to be confusion in load_from and pretrained parameters in config files.
In [https://github.com/open-mmlab/mmaction2/blob/28ac80cb1a96c208a3e95f76f051ebafc74390aa/configs/recognition/slowonly/slowonly_imagenet_pretrained_r50_8x4x1_64e_ucf101_rgb.py](url) slowonly model uses only imagenet pretrained model, while in [https://github.com/open-mmlab/mmaction2/blob/28ac80cb1a96c208a3e95f76f051ebafc74390aa/configs/recognition/slowonly/slowonly_k400_pretrained_r50_8x4x1_40e_ucf101_rgb.py](url) model uses Kinetics400 pretrained model but if we look into the log file for the second experiment [https://download.openmmlab.com/mmaction/recognition/slowonly/slowonly_k400_pretrained_r50_8x4x1_40e_ucf101_rgb/20210606_010231.log](url) we find that the model is first loading (load model from: torchvision://resnet50) then in the next couple of lines its again loading (load checkpoint from https://download.openmmlab.com/mmaction/recognition/slowonly/slowonly_r50_8x8x1_256e_kinetics400_rgb/slowonly_r50_8x8x1_256e_kinetics400_rgb_20200703-a79c555a.pth).

So is it the case for example, that for using Kinetics400 pretrained model we first load the ImageNet pretrained weights and then load the Kinetics400 pretrained checkpoint.

If it is not the case then why are you loading 2 checkpoints one after another.
Please clarify.

Thank You.

**Comment by hukkai** [comment]
Yes it is not neccesary to  load 2 checkpoints one after another.
